### ISSUE TITLE
Add medulloPackage as analysis method

### ISSUE THREAD

**Issue by envest** [AlexsLemonade/medulloblastoma-classifier#63]
https://github.com/d3b-center/medullo-classifier-package can now work as a single sample predictor -- we need to include it as a comparison method and understand its functionality

Once Single Sample Classification PR (#6) is merged:
- implement `test_medullo()` in `utils/modeling.R` with `run_one_model()` and `run_many_models()`
- update documentation throughout repo
- update each notebook to include medullo comparison

[Rathi, K. S. et al. A transcriptome-based classifier to determine molecular subtypes in medulloblastoma. PLoS Comput. Biol. 16, e1008263 (2020)](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008263)

**Comment by envest** [comment]
One concern about using this method is training data leakage. Many (all?) of the data sets used to train medullo also appear in our study. We need to be intentional about limiting our medullo testing to data sets not appearing in their training set. This may take the form of a distinct test analysis on particular data sets that don't overlap with their study.

**Comment by jaclyn-taroni** [comment]
I believe the only _training_ datasets are:

* [`GSE37418`](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37418)
* [`EGAD00001001899`](https://ega-archive.org/datasets/EGAD00001001899), which notably does not appear to have a count or abundance matrix on GEO (the only GEO Series associated with [the publication](https://pmc.ncbi.nlm.nih.gov/articles/PMC4794752/) is from mouse).

**Comment by envest** [comment]
Awesome -- glad to be wrong about this!

For each repeat, we split our training/testing sets at the _project_ level. In the baseline model code as of now, half of the projects from each platform go to training, and the remainder go testing. We should be able to eliminate any repeat where `GSE37418` samples appear in the test set.

**Comment by jaclyn-taroni** [comment]
We discussed two options for approaching this:

1. Add some logic that skips `medulloPackage` if `GSE37418` is in the test data.
2. Always force `GSE37418` to be in the training data for all the repeats, which should be feasible since we have many more microarray studies than RNA-seq studies and it is HGU133plus2.

**Comment by jaclyn-taroni** [comment]
* @envest to always force `GSE37418` to be in the training data
* @jaclyn-taroni to implement `test_medullo()` in `utils/modeling.R` with `run_one_model()` and `run_many_models()`

**Comment by envest** [comment]
I have forced GSE37418 to always be in the training data... will implement regenerating the models tomorrow and then file PR of branch `envest/always_train_with_GSE37418`.

Would also note that train/test study information is currently captured in each repeat's metadata -- this data frame contains training set metadata for the first repeat and includes a "study" column: ```baseline_models[[1]]$train_metadata```
